README.md

```diff
@@ -0,0 +1,701 @@
+
+### `from_timestamp`
+
+Including `from_timestamp` restricts a capabilities authority to only operations' which contain a greater `timestamp` in their header.
+
+```json
+{
+  //...
+  "action": "document/update",
+  "conditions": {
+    "document_ids": ["0A01.."],
+    "from_timestamp": 1712226632
+  }
+}
+```
+
+The above capability restricts authority to document `"0A01"` and only operations containing a timestamp `>` `1712226632`.
+
+### `to_seq`
+
+Including `to_seq` restricts a capabilities authority to only operations' which contain a lower `timestamp` in their header.
+
+An authors contributions to a document contain a monotonically increasing sequence number. By leveraging this property we can introduce a more strictly bounded authority than is possible when using timestamps.
+
+```json
+{
+  //...
+  "action": "document/update",
+  "conditions": {
+    "document_ids": ["0A01.."],
+    "to_seq": 100
+  }
+}
+```
+
+The above capability restricts authority to document `"0A01"` and only operations containing a sequence number `<` 100. Effectively this capability allows a maximum of 100 operations to be published to a document.
+
+### `from_seq`
+
+Including `from_seq` restricts a capabilities authority to only operations' which contain a greater `timestamp` in their header.
+
+### `not_before`
+
+The UTC Unix Timestamp in milliseconds specifying at which point this capability becomes valid.
+
+### `expires`
+
+The UTC Unix Timestamp in milliseconds specifying at which point this capability expires.
+
+Although this field is optional, it is highly recommended that one includes an expiry for most capabilities. The exception for this is when issuing capabilities containing a restriction based on an operations sequence number. As this is a securely bounded condition, with measures in place at a protocol level to detect potentially malicious behavior, then expiry can be safely omitted.
+
+# Issuing
+
+The owner of a resource can issue a root capability which grants authority to the receiver based on the action and conditions.
+
+```mermaid
+classDiagram
+    Capability --> Anna : issuer
+    Capability --> Billie : receiver
+    Conditions --> Capability
+    Capability --> Document: described by conditions
+    class Document {
+      id = "0A01"
+      author = Anna
+    }
+    class Conditions {
+      document_ids = ["0A01"]
+    }
+    class Capability {
+      issuer = Anna
+      receiver = Billie
+      subject = Anna
```

Comment:
authRoot? issuerRoot

i keep thinking subject is eg a document. I think the name could be clearer, but you may be going with other  conventions ?

Reply:
Good feedback, thanks :+1: